kuryr_kubernetes/controller/drivers/lbaasv2.py

```diff
@@ -839,8 +845,12 @@ def _find_loadbalancer(self, loadbalancer):
             loadbalancer.port_id = self._get_vip_port(loadbalancer).get("id")
```

Comment:
Can you also fix this? I am not sure if this could cause issues when cleaning LoadBalancer, if it for some reason doesn't have a VIP port it  will end up with Traceback.

```
        vip_port = self._get_vip_port(loadbalancer)
        if vip_port:
            loadbalancer.port_id = vip_port.get("id")
```

Reply:
Yeah so if the neutron port wouldn't exist (would be deleted for any reason, and the LB is in ERROR state) this would return None as it would fail on getting the port_id from neutron. Probably we should still delete ERROR state LBs without vip_port.

Reply:
How about I just moved checking for ERROR above before we try looking for the VIP port?

Reply:
That won't work, what if the LB is moved from ERROR state to PENDING_DELETE? That will cause the same issue.